autoreject/autoreject.py

```diff
@@ -44,8 +60,15 @@ def _slicemean(obj, this_slice, axis):
     return mean
 
 
-def validation_curve(epochs, y=None, param_name="thresh", param_range=None,
-                     cv=None, return_param_range=False, n_jobs=1):
+def validation_curve(
+    epochs,
+    y=None,
+    param_name="thresh",
+    param_range=None,
+    cv=None,
+    return_param_range=False,
+    n_jobs=1,
+):
```

Comment:
I know some automatic formatting tools do this but I actually find the original version more readable. Is it possible to easily revert to that version? It will also make it slightly easier for me to see what are the actual code changes independent of formatting changes.